lib/types/index.js

```diff
@@ -503,6 +503,7 @@ export type Project = {
   autoFetchHour: number,
   autoFetchMinute: number,
   bounds?: Bounds,
+  defaultLanguage: string,
```

Comment:
I think @abyrd and I removed the `defaultLanguage` field a year or so ago: https://github.com/catalogueglobal/datatools-server/commit/c072b0f9ef079f533fa25189f10f6953045d9fd2

It was only ever used to pre-populate the agency and feed_info language fields, but this is not really implemented in the current version and the field is basically unneeded. I only noticed this wasn't in the server because the field is not saving properly in the UI. But I think the best thing to do is just remove the field entirely from the UI.